src/Ookii.Dialogs.Wpf/ProgressDialog.cs

```diff
@@ -59,6 +59,8 @@ private class ProgressChangedData
         private bool _cancellationPending;
         private int _percentProgress;
         private IntPtr _ownerHandle;
+        private CancellationToken _cancellationToken;
```

Comment:
Let's remove this as we rely on the `CancellationToken` stored in the `_cancellationSource`